### ISSUE TITLE
Feature request - Add support for non-color variables

### ISSUE THREAD

**Issue by AcelisWeaven** [wtho/bulma-css-vars#4]
One feature I'd like to see would be the possibility to change other properties, like the font-family.

The ColorUpdater wouldn't be needed, and you could simply do something like this:
```js
    document.documentElement.style.setProperty('--body-family', 'OpenSans, sans-serif');
```

However, I don't know if it fits the project. What's your opinion about this?

**Comment by wtho** [comment]
Yeah, for fonts there is no need to provide additional wrappers like it is for colors. I think it is clearer to set the css variables directly in your code.

I identified two more cases where computing derived variables requires extra work:
* calculations computed in sass (like `$level-item-spacing: ($block-spacing / 2)` in [`level.sass`](https://github.com/jgthms/bulma/blob/master/sass/components/level.sass)), which either have to be redone in CSS using `calc` or have to be done using js when changing the css variable
* The `percentage` function used in [`columns.sass`](https://github.com/jgthms/bulma/blob/master/sass/grid/columns.sass)

But I think it's worthwhile waiting until someone requires this feature.
